iai-callgrind-runner/src/main.rs

```diff
@@ -70,19 +70,20 @@ fn main() {
                 IaiCallgrindError::VersionMismatch(cmp, runner_version, library_version) => {
                     match cmp {
                         Cmp::Lt => error!(
-                            "iai-callgrind-runner ({}) is older than iai-callgrind ({}). Please \
-                             update iai-callgrind-runner",
+                            "iai-callgrind-runner ({0}) is older than iai-callgrind ({1}). Please \
+                             update iai-callgrind-runner by calling 'cargo install --version {1} \
+                             iai-callgrind-runner'",
                             runner_version, library_version
                         ),
                         Cmp::Gt => error!(
-                            "iai-callgrind-runner ({}) is newer than iai-callgrind ({}). Please \
-                             update iai-callgrind",
+                            "iai-callgrind-runner ({0}) is newer than iai-callgrind ({1}). Please \
+                             update iai-callgrind to '{1}' in your Cargo.toml file",
```

Comment:
iai-callgrind needs to be updated to the newer runner version, which is `{0}`. So, this needs to be `... update iai-callgrind to '{0}' in your ...`

Reply:
Thanks :)